screens/HomeScreen.jsx

```diff
@@ -26,7 +26,7 @@ export default function HomeScreen({ navigation }) {
   }, []);
 
   function handlePress(date) {
-    navigation.navigate('Goal');
+    // setProfile({...profile, goalHistory: [...goalHistory, date]})
```

Comment:
What will happen here?

Reply:
not sure yet :)  just need to figure out how to add the dates to state so they can be displayed on the home page